support-frontend/app/controllers/WeeklySubscriptionController.scala

```diff
@@ -43,14 +41,14 @@ class WeeklySubscriptionController(
   def weekly(countryCode: String, orderIsAGift: Boolean): Action[AnyContent] = CachedAction() { implicit request =>
     implicit val settings: AllSettings = settingsProvider.getAllSettings()
     val canonicalLink = Some(buildCanonicalWeeklySubscriptionLink("uk", orderIsAGift))
-    val defaultPromo = List(ausElection)
 
     val queryPromos =
       request.queryString
-        .get("promoCode")
-        .map(_.toList)
-        .getOrElse(defaultPromo)
-    val maybePromotionCopy = landingCopyProvider.promotionCopy(queryPromos, GuardianWeekly, countryCode)
+        .getOrElse("promoCode", Nil)
+        .toList
+    val defaultPromos = priceSummaryServiceProvider.forUser(false).getDefaultPromoCodes(GuardianWeekly)
```

Comment:
hardly worth a commit for, but thought I'd mention it anyway
```suggestion
    val defaultPromos = priceSummaryServiceProvider.forUser(isTestUser = false).getDefaultPromoCodes(GuardianWeekly)
```

Reply:
👍